### PR TITLE
PMM-7338 API test for change check interval

### PR DESCRIPTION
**Pull request by palash25**
https://github.com/percona/pmm/pull/714
https://github.com/Percona-Lab/pmm-submodules/pull/1644
https://github.com/percona/pmm-managed/pull/712

### CODE REVIEW

**Review by BupycHuk — CHANGES_REQUESTED** [review]
Please update tests.

File: server/checks_test.go [hunk 1]
```diff
@@ -130,49 +130,155 @@ func TestListSecurityChecks(t *testing.T) {
...
 		require.NoError(t, err)
 		require.NotEmpty(t, resp.Payload.Checks)
+		assert.Equal(t, "STANDARD", string(*resp.Payload.Checks[0].Interval))
 
-		assert.Equal(t, check.Name, resp.Payload.Checks[0].Name)
-		assert.Equal(t, !check.Disabled, resp.Payload.Checks[0].Disabled)
-	}
+		var params *security_checks.ChangeSecurityChecksParams
+		interval := "RARE"
+		for i, check := range resp.Payload.Checks {
```
[the remark this answers is not in the document]
Comment (palash25) [reply server/checks_test.go:239]: will fix it

**Review by artemgavrilov — APPROVED** [review]

File: server/helpers.go [hunk 2]
```diff
@@ -46,3 +48,31 @@ func restoreSettingsDefaults(t *testing.T) {
...
+	var params *security_checks.ChangeSecurityChecksParams
+
+	interval := "STANDARD"
+	for _, check := range resp.Payload.Checks {
+		params = &security_checks.ChangeSecurityChecksParams{
+			Body: security_checks.ChangeSecurityChecksBody{
+				Params: []*security_checks.ParamsItems0{
+					{
+						Name:     check.Name,
+						Interval: &interval,
```
Comment (BupycHuk) [inline server/helpers.go:68]:
```suggestion
						Interval: pointer.ToString(security_checks.ParamsItems0IntervalSTANDARD),
```

File: server/checks_test.go [hunk 3]
```diff
@@ -130,49 +130,160 @@ func TestListSecurityChecks(t *testing.T) {
...
+		var params *security_checks.ChangeSecurityChecksParams
+		interval := "RARE"
+		// convert all checks to RARE interval
+		for _, check := range resp.Payload.Checks {
+			params = &security_checks.ChangeSecurityChecksParams{
+				Body: security_checks.ChangeSecurityChecksBody{
+					Params: []*security_checks.ParamsItems0{
+						{
+							Name:     check.Name,
+							Interval: &interval,
```
Comment (BupycHuk) [inline server/checks_test.go:250]:
```suggestion
							Interval: pointer.ToString(security_checks.ParamsItems0IntervalSTANDARD),
```

File: server/checks_test.go [hunk 4]
```diff
@@ -130,49 +130,160 @@ func TestListSecurityChecks(t *testing.T) {
...
+		})
 		require.NoError(t, err)
+		assert.True(t, res.Payload.Settings.SttEnabled)
+
+		resp, err := managementClient.Default.SecurityChecks.ListSecurityChecks(nil)
+		require.NoError(t, err)
+		require.NotEmpty(t, resp.Payload.Checks)
+		assert.Equal(t, "STANDARD", string(*resp.Payload.Checks[0].Interval))
+
+		var params *security_checks.ChangeSecurityChecksParams
```
Comment (BupycHuk) [inline server/checks_test.go:241]: let's remove these two variables from here and replace `params =` with `params :=` below

Comment (BupycHuk) [reply server/checks_test.go:241]: here and in other places where it's possible

**Review by BupycHuk — APPROVED** [review]
A few comments, please fix them and merge

**Comment by palash25** [comment]
done

**Pull request merged**